api/swagger/swagger.yaml

```diff
@@ -294,6 +294,23 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /sources:
+    x-swagger-router-controller: sources
+    get:
+      tags:
+        - sources
+      description: Returns list of sources
+      operationId: list
```

Comment:
In the same time, swagger disagrees with reused `operationId`s and outputs red errors in the editor if it finds more of the same.
It also has a point in being able to identify the operations by ID if needed, hence the name. The main idea is, actually, that we should not reuse `operationId`s, as we're not reusing IDs in any of the other specifications we're using (databases, markup etc.)